### DISCUSSION TITLE
Hide the VM "nature" to Windows and apps

### DISCUSSION THREAD

**Discussion by freechelmi** [winapps-org/winapps#46] [Support]
A lot of apps and games ( well mostly games ? )  won't start if they detect that windows is running in a VM.

I've read mixed ideas and results on how to achieve this 

The first thing seems to avoid to install Virtio drivers which can really be a problem.

Do you have any working solution to this ?

**Comment by oskardotglobal** [comment]
Well, quickemu uses qemu too, so if we figure out how to mask that inside QEMU then we can do it in quickemu as well
I don't know how to do this yet, but I think it'd be easy when doing some research

**Comment by LDprg** [comment]
Our best bet is probaply to wait or search for a software where this problem accurs. Not using virtio feels like not an option, because well these drivers are crazy in terms of speed and efficiency.

However you can hide it by configuring the libvirt files. https://superuser.com/questions/1387935/hiding-virtual-machine-status-from-guest-operating-system#1389159
